tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -252,7 +256,8 @@
             # These vars are for the network config templates themselves and are
             # considered EDPM network defaults.
             neutron_physical_bridge_name: br-ctlplane
-            neutron_public_interface_name: eth0
+            neutron_public_interface_name: {{ dataplane_public_iface | default('eth0') }}
```

Comment:
might still be some issue with these values.

the tests are passing but we are losing connectivity to the standalone over the main interface (eth0). i can ssh into it from the other nodes e.g. crc node using the 192.168 address.

Reply:
the problem i think is that I want to pass routes as []

but this is being treated as 'empty' and defaulting to the ctlplane_host_routes

Reply:
yes using the boolean flag for setting the route seems to work in last test